packages/mobile/src/screens/change-email-screen/ChangeEmailScreen.tsx

```diff
@@ -0,0 +1,184 @@
+import { useCallback, useEffect } from 'react'
+
+import {
+  ChangeEmailPage,
+  useChangeEmailFormConfiguration
+} from '@audius/common/hooks'
+import type {
+  EventListenerCallback,
+  EventMapCore,
+  NavigationState
+} from '@react-navigation/native'
+import { createNativeStackNavigator } from '@react-navigation/native-stack'
+import { Formik, useFormikContext } from 'formik'
+import { TouchableOpacity } from 'react-native'
+
+import { IconArrowRight, IconClose } from '@audius/harmony-native'
+import { BackButton } from 'app/app/navigation/BackButton'
+import {
+  Button,
+  KeyboardAvoidingView,
+  ModalScreen,
+  Screen,
+  ScreenContent
+} from 'app/components/core'
+import { useNavigation } from 'app/hooks/useNavigation'
+import { useToast } from 'app/hooks/useToast'
+import { makeStyles } from 'app/styles'
+
+import { useAppScreenOptions } from '../app-screen/useAppScreenOptions'
+import { VerifyEmailSubScreen } from '../change-password-screen/SubScreens'
+
+import { ConfirmPasswordSubScreen, NewEmailSubScreen } from './SubScreens'
+
+const messages = {
+  change: 'Change Email',
+  continue: 'Continue',
+  success: 'Email updated!'
+}
+
+const useStyles = makeStyles(({ palette, spacing }) => ({
```

Comment:
guessing this is a copy and youre aware of the style={css({})} pattern?

Reply:
I don't think that exists on `Screen`?

Reply:
or `KeyboardAvoidingView`